frontend/src/containers/DateFormatSettings.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { useSettings } from "../hooks";
+import dayjs from "dayjs";
+import SettingsLayout from "../layouts/Settings";
+import Button from "../components/Button";
+import AlertContainer from "./AlertContainer";
+import "./PublishedSiteSettings.css";
+
+function DateFormatSettings() {
```

Comment:
I think here and in the Publishing Guidance settings page we need to add a spinner if settings is not already loaded. Now it is loaded the default and then it changes to the settings value.

Reply:
Ok, let me try to add a loading flag. I wasn't sure if it made sense because Settings are loaded only once now.